include/utility.hpp

```diff
@@ -310,21 +316,8 @@ struct TEnumInfo {
 };
 
 template <typename E, bool IsFlags = false> struct TEnumInfoImpl {
-  static constexpr auto enum_names() {
-    if constexpr (IsFlags) {
-      return magic_enum::flags::enum_names<E>();
-    } else {
-      return magic_enum::enum_names<E>();
-    }
-  }
-
-  static constexpr auto enum_values() {
-    if constexpr (IsFlags) {
-      return magic_enum::flags::enum_values<E>();
-    } else {
-      return magic_enum::enum_values<E>();
-    }
-  }
+  static constexpr auto enum_names() { return magic_enum::enum_names<E>(); }
+  static constexpr auto enum_values() { return magic_enum::enum_values<E>(); }
```

Comment:
Are you sure about that?
I remember I had issues with it hence why this code was changed.

Reply:
It was added for enum flags with ImGui which we don't have anymore. So we will see once we have a need for new ones.